ci-operator/step-registry/assisted/equinix/assisted-equinix-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: assisted-equinix
+  steps:
+    cluster_profile: packet-assisted
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    pre:
+      - ref: assisted-equinix-setup
+      - chain: assisted-common-pre
+    test:
+      - ref: assisted-baremetal-conf
+      - ref: assisted-baremetal-test
```

Comment:
(not necessarily in the scope of this PR) in the future we might want to move to ``assisted-common-verify``

Reply:
Yes, we already have a ticket for that (assigned to me :see_no_evil: )